### PR TITLE
Update r10k mcollective agent to work with Choria

### LINKED ISSUE #431: Migrate r10k-mcollective to choria
The r10k-mcollective plugin that is provided by this module is limiting in that it requires the entire r10k module to utilize. As some users may want to use the mcollective plugin without the full r10k module AND to move the plugin to a centralized location better suited to maintaining Mcollective plugins, we are going to work with @ripienaar to migrate the plugin to a standalone module under the [Choria](https://github.com/choria-io) namespace.

### PR DESCRIPTION
**Pull request by treydock**
#### Pull Request (PR) description

Change how the mcollective agent is deployed to work with Choria.  The updates to fixtures are a bit large due to number of dependencies needed to compile catalog when adding a mcollective plugin.  The mcollective agent modules are not actually git repos but puppet modules generated for the forge from the source code so only way to get the modules is from the forge.

#### This Pull Request (PR) fixes the following issues

Fixes #431

### CODE REVIEW

File: files/mcollective/agent/r10k.json [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+          "description": "Error from git",
+          "display_as": "Errors",
+          "default": null
+        }
+      },
+      "display": "always",
+      "description": "Synchronize "
+    }
+  ]
+}
```
Comment (bastelfreak) [inline files/mcollective/agent/r10k.json:308]: can you please add the missing newline at the end?

File: .fixtures.yml [hunk 2]
```diff
@@ -6,3 +6,21 @@ fixtures:
...
+      puppet_version: ">= 6.0.0"
+    apt: "https://github.com/puppetlabs/puppetlabs-apt"
+  forge_modules:
+    mcollective_choria: "choria/mcollective_choria"
+    mcollective_agent_puppet: "choria/mcollective_agent_puppet"
+    mcollective_agent_service: "choria/mcollective_agent_service"
+    mcollective_agent_package: "choria/mcollective_agent_package"
+    mcollective_agent_filemgr: "choria/mcollective_agent_filemgr"
+    mcollective_util_actionpolicy: "choria/mcollective_util_actionpolicy"
+  symlinks:
```
Comment (bastelfreak) [inline .fixtures.yml:25]: the `symlinks` hash isn't required anymore. You can omit that.

File: files/mcollective/agent/r10k.json (hunk 1 above)
Comment (treydock) [reply files/mcollective/agent/r10k.json]: Fixed

File: .fixtures.yml (hunk 2 above)
Comment (treydock) [reply .fixtures.yml]: Removed `symlinks`

**Comment by bastelfreak** [comment]
this should remove the failing tests: https://github.com/voxpupuli/puppet-r10k/pull/549

**Comment by bastelfreak** [comment]
@treydock can you please against our latest master?

**Review by bastelfreak — APPROVED** [review]

**Review by ghoneycutt — APPROVED** [review]

**Pull request merged**